packages/component-base/src/resize-mixin.js

```diff
@@ -8,6 +8,10 @@ import { dedupingMixin } from '@polymer/polymer/lib/utils/mixin.js';
 const observer = new ResizeObserver((entries) => {
   setTimeout(() => {
     entries.forEach((entry) => {
+      if (!entry.target.isConnected) {
+        return;
+      }
```

Comment:
Is this change related? Couldn't find which package is affected

Reply:
It seems some tests finish before the resize observer processes the changes they triggered. If the size observer then triggers a Lit update, Lit throws an exception because the element no longer exists by that moment.